bin/rtdev-setup-releases.sh

```diff
@@ -39,20 +39,16 @@ cd $RT_DEST_DIR
 git init 
 
 ## Some versions of git and/or OS require these fields
-HAS_LOCAL=$(git config --local 2>&1 | grep unknown)
-if [ -z "$HAS_LOCAL" ]; then
-    git config --local user.name "Riak Test"
-    git config --local user.email "dev@basho.com"
-    git config --local core.autocrlf input
-    git config --local core.safecrlf false
-    git config --local core.filemode true
-fi
+git config --local user.name "Riak Test"
```

Comment:
Can you please revert these changes to this script? It is supposed to work on different versions of git now